docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
```

Comment:
@pestOO we have no global plugin manager. After merging RevSDL we have Plugin manager that is related to Commands, but it is not mentioned in it's name